### DISCUSSION TITLE
Is it possible to run AWS lambda functions using dotnet sdk docker image

### DISCUSSION THREAD

**Discussion by zeusongit** [dotnet/dotnet-docker#4339] [Q&A]
Hey, so I have a use-case where I have an executable that requires .NET6 SDK and the lambda function should trigger that executable with the required arguments to run it inside lambda, but the aws lambda runtime for .NET6 provide by Amazon does not include the SDK part to run the executable. Is it possible to run the Lambda function using the .NET6 sdk image?

**Comment by normj** [comment]
@zeusongit I just answered your question for this on our AWS Lambda repository https://github.com/aws/aws-lambda-dotnet/discussions/1413
